platform-modules/content-manager/src/main/java/org/ekstep/content/operation/finalizer/ReviewFinalizer.java

```diff
@@ -255,7 +255,8 @@ private void validateResource(String collectionId){
 				notPublishedList.add(resource.getIdentifier());
 			}
 		}
-		if(CollectionUtils.isNotEmpty(notPublishedList)){
+		Boolean childPublishEnabled = Platform.config.getBoolean("child.publish.enable");
```

Comment:
```suggestion
		Boolean childPublishEnabled = Platform.getBoolean("collection.only_published_children", true);
```

Reply:
org.ekstep.common.Platform doesnot contain the getBoolean(String, Object) method, So not able to address this.